charts/grist/examples/helmfile/local-cluster.sh

```diff
@@ -88,5 +90,7 @@ data:
 EOF
 
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-kubectl -n ingress-nginx create secret tls mkcert --key 127.0.0.1.nip.io+1-key.pem --cert 127.0.0.1.nip.io+1.pem
+if ! kubectl -n ingress-nginx get secret mkcert; then
```

Comment:
Shouldn't the output be discarded? As I understand the code above, if the secret exist, it will be listed.

```suggestion
if ! kubectl -n ingress-nginx get secret mkcert &>/dev/null; then
```

Though that's not a big deal.

Reply:
The output is 
```bash
kubectl -n ingress-nginx get secret mkcert
NAME     TYPE                DATA   AGE
mkcert   kubernetes.io/tls   2      100m
```

So not much secret leaked :)

But in order of not logging noise i'm ok with your suggestion :)